packages/docs/open-source/plugins/plugin-docusaurus.md

```diff
@@ -4,31 +4,76 @@ outline: deep
 
 # Docusaurus Plugin
 
-The plugin-docusaurus plugin allows you to index the content of your Docusaurus websites and offer text search to your visitors.
+The docusaurus plugin allows you to index the content of your Docusaurus websites and offer text search to your visitors.
+This plugin can be used in OSS mode where you host the documents, or in cloud mode where your data will be stored by us and you will have access to analytics where you can see the number of queries and much more.
 
 ## Pre-requisites
-In order guarantee a correct functionality of the plugin, you need to have the `@docusaurus/core` at least in the version `2.4.3`.
+
+In order guarantee a correct functionality of the plugin, you need to have the `@docusaurus/core` at least in the version `3.0.0`.
```

Comment:
The minimum required docusaurus/core version is set to 3.2.0

Reply:
fixed!